code/_onclick/hud/alert.dm

```diff
@@ -40,7 +40,8 @@
 		var/old_plane = new_master.plane
 		new_master.layer = FLOAT_LAYER
 		new_master.plane = FLOAT_PLANE
-		alert.overlays += new_master
+		var/atom/yes_this_is_a_bloody_atom = alert
+		yes_this_is_a_bloody_atom.add_overlay(new_master)				// B Y O N D
```

Comment:
what are you doing, this does nothing you know?
```/obj/screen/alert/alert```, it's an ```/obj```, which is itself an ```/atom/movable```

If you're attempting to cast alert to atom so as to only call the atom version of ```add_overlay()```, spoilers, DM doesn't do that.